CHANGELOG.md

```diff
@@ -4,6 +4,54 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 0.11.0 (2018-08-08)
+
+**High-level changes**
+
+* Hardening the database: blocking commands that do not make sense for hypertables, adding support for several previously unsupported table alterations, adding more test coverage.
+* **Initial adaptive chunking** support (default is off). Allows users to specify a target chunk size (in bytes) and then automatically updates chunk intervals to match target.
+* Support for additional types of time columns, if they are binary compatible (thanks @fvannee!).
+
```

Comment:
**Major improvements**

* **Adaptive chunking**.  This feature, currently in beta, allows the database to automatically adapt a chunk's time interval, so that users do not need to manually set (and possibly manually change) this interval size.  In this release, users can specify either a target chunk data *size* (in terms of  MB or GB), and the chunk's time intervals will be automatically adapted.  Alternatively, users can ask the database to just estimate a target size itself based on the platform's available memory and other parameters, and the system will adapt accordingly.  This type of automation can simplify initial database test and operations.  This feature is default off.  *Note:  The default time interval for non-adaptive chunking has also been changed from 1 month to 1 week.*

* **Continued hardening**:  This release addresses a number of less frequently used schema modifications, functions, or constraints.  Unsupported functions are safely blocked, while we have added support for a number of new types of table alterations.  This release also adds additional test coverage.